CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Coming soon:
 - Support for auto-instrumentation configuration for the MetricsExporter.
 
+## [0.5.0] - 2020-06-08
+- Support for OpenTelemetry-Java version 0.5.0
+
```

Comment:
#26 is user-facing and included.  Also #53 right?  Worth mentioning I think.

Reply:
will update! good catch!

Reply:
done. ptal